sandboxed-fs.js

```diff
@@ -4,11 +4,109 @@ const sandboxedFs = {};
 module.exports = sandboxedFs;
 
 const fs = require('fs');
+const pathModule = require('path');
+
+const errorMessage = 'path must be a string';
```

Comment:
And it looks like even `fs` itself isn't being consistent with this.

![image](https://cloud.githubusercontent.com/assets/4923335/23089852/6766b636-f599-11e6-80b7-eb38e23129b0.png)